checker/src/org/checkerframework/checker/nullness/NullnessVisitor.java

```diff
@@ -424,16 +442,33 @@ public Void visitTypeCast(TypeCastTree node, Void p) {
     // ///////////// Utility methods //////////////////////////////
 
     /**
-     * Issues a 'dereference.of.nullable' if the type is not of a {@link NonNull} type.
+     * Issues the error message if the type is not of a {@link NonNull} type.
      *
      * @param tree the tree where the error is to reported
```

Comment:
'is to reported' -> 'is to be reported' or 'should be reported'? Same in the copy below.